src/services/completions.ts

```diff
@@ -1527,58 +1528,51 @@ namespace ts.Completions {
          * Relevant symbols are stored in the captured 'symbols' variable.
          */
         function tryGetClassLikeCompletionSymbols(): GlobalsSearch {
-            const classLikeDeclaration = tryGetClassLikeCompletionContainer(contextToken);
-            if (!classLikeDeclaration) return GlobalsSearch.Continue;
+            const decl = contextToken && tryGetObjectTypeDeclarationCompletionContainer(contextToken, location);

```

Comment:
this seems like change from previous behaviour where event if contextToken was not present we would use location to decide on class location. Why is this changed ?